cloudlift/deployment/service_template_generator.py

```diff
@@ -342,6 +442,94 @@ def _gen_log_config(self, service_name):
             }
         )
 
+    def _add_nlb(self, cd, service_name, config, launch_type):
```

Comment:
there is no much difference  how we create nlb, vs alb. need lot of refactoring here as well.